File: src/main/java/com/lessmarkup/engine/language/XmlLanguageFile.java
/*
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */
package com.lessmarkup.engine.language;

import java.util.ArrayList;
import java.util.List;

public class XmlLanguageFile {
    private String name;
    private String id;
    private String shortName;
    private final List<XmlTranslation> translations = new ArrayList<>();

    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }

    public String getId() {
        return id;
    }
    public void setId(String id) {
        this.id = id;
    }

    public String getShortName() {
        return shortName;
    }
    public void setShortName(String shortName) {
        this.shortName = shortName;
    }

    public List<XmlTranslation> getTranslations() {
        return translations;
    }
}

File: src/main/scala/com/lessmarkup/engine/language/XmlLanguageFile.scala
/*
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

package com.lessmarkup.engine.language

class XmlLanguageFile {
  private var name: String = null
  private var id: String = null
  private var shortName: String = null
  private final val translations: java.util.List[XmlTranslation] = new java.util.ArrayList[XmlTranslation]

  def getName: String = name

  def setName(name: String) {
    this.name = name
  }

  def getId: String = id

  def setId(id: String) {
    this.id = id
  }

  def getShortName: String = shortName

  def setShortName(shortName: String) {
    this.shortName = shortName
  }

  def getTranslations: java.util.List[XmlTranslation] = {
    translations
  }
}